src/platform/testing/e2e/cypress/support/commands/expandAccordions.js

```diff
@@ -1,13 +1,53 @@
 /**
- * Expands all accordions and AdditionalInfo components.
+ * Expands all Accordions and AdditionalInfo components.
+ * Web Components that require Shadow DOM broken out from React Components
  */
 Cypress.Commands.add('expandAccordions', () => {
   Cypress.log();
 
   cy.get('main').then($main => {
-    const accordions = $main.find('button[aria-expanded=false]');
-    if (accordions.length) {
-      cy.wrap(accordions).each(button => {
+    // Check if va-accordion-item Web Component exists
+    if ($main.find('va-accordion-item').length > 0) {
+      cy.get('va-accordion-item')
+        .shadow()
+        .then(accordion => {
+          // If it exists and Accordions are not already expanded
+          if (accordion.find('button[aria-expanded=false]').length > 0) {
+            cy.get('va-accordion-item')
+              .shadow()
+              .find('button[aria-expanded=false]')
```

Comment:
I've got two questions here:

- Is the `if` on line 15 necessary with the `find` in place on line 18?
- What is the `cy.get('va-accordion-item')` on line 16 giving us that we don't aleady have with the `accordion` param?